examples/systemd/machine-id/README.md

```diff
@@ -0,0 +1,42 @@
+# Systemd Service
+
+Create and fill out a configuration file for Machine ID at `/etc/tbot.yaml`.
+
+```yaml
+auth_server: "auth.example.com:3025"
+onboarding:
+  join_method: "token"
+  token: "00000000000000000000000000000000"
+  ca_pins:
+  - "sha256:1111111111111111111111111111111111111111111111111111111111111111"
+storage:
+  directory: /var/lib/teleport/bot
+destinations:
+  - directory: /opt/machine-id
+```
+
+Next initialize ownership of the short-lived certificate directory. In this
+example, ownership will belong to the user:group `jenkins:jenkins`. Make sure
+the `jenkins` user exists on your system.
+
+```bash
+$ tbot init \
+    --destination-dir=/opt/machine-id \
+    --owner=jenkins:jenkins
```

Comment:
should we recommend `jenkins` as the owner of the tbot service?

Reply:
I just removed `tbot init` from this example. This is more advanced functionality that can be used after reading the docs.